runtime/drivers/druid/druid_api_sql_driver.go

```diff
@@ -0,0 +1,354 @@
+package druid
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type druidSQLDriver struct{}
+
+var _ driver.Driver = &druidSQLDriver{}
+
+func (a *druidSQLDriver) Open(dsn string) (driver.Conn, error) {
+	client := http.Client{Timeout: time.Second * 10}
+
+	return &sqlConnection{
+		client: &client,
+		dsn:    dsn,
+	}, nil
+}
+
+func init() {
+	sql.Register("druid", &druidSQLDriver{})
+}
+
+type sqlConnection struct {
+	client *http.Client
+	dsn    string
+}
```

Comment:
A Druid driver for `database/sql` should be a standalone component, so can we put this in `runtime/drivers/druid/druidsqldriver` or something like that?